src/message_bus.zig

```diff
@@ -730,6 +730,12 @@ fn MessageBusType(comptime process_type: vsr.ProcessType) type {
                 @memcpy(mem.asBytes(&header), data[0..@sizeOf(Header)]);
 
                 if (!connection.recv_checked_header) {
+                    if (header.command == .reserved) {
+                        log.err("reserved command received from {}", .{connection.peer});
+                        connection.terminate(bus, .shutdown);
+                        return null;
+                    }
+
```

Comment:
We should check this _after_ we validate the checksum, to make sure it is really reserved, rather than just a packet corruption